### ISSUE TITLE
stringifyRecordValues and stringifyObjectValues libs demand to be default export by Webpack

### ISSUE THREAD

**Issue by sjosuerojas** [bootstrap-vue/bootstrap-vue#6578]
### These stringifyRecordValues and stringifyObjectValues libs appeared to be exported as a normal function without the -default- key name

Hi There! I'm doing a Vue project without using the vue-cli service intead I'm using wepack 5 and I says the following warnings:

WARNING in ./node_modules/bootstrap-vue/esm/components/table/helpers/default-sort-compare.js 68:9-30
export 'stringifyObjectValues' (imported as 'stringifyObjectValues') was not found in '../../../utils/stringify-object-values' (possible exports: default)
 @ ./node_modules/bootstrap-vue/esm/components/table/helpers/mixin-sorting.js 17:0-60 98:21-39
 @ ./node_modules/bootstrap-vue/esm/components/table/table.js 25:0-78 33:471-483 43:32-44
 @ ./node_modules/bootstrap-vue/esm/index.js 249:0-50 249:0-50
 @ ./src/app.js 3:0-58 8:8-20 10:8-19

WARNING in ./node_modules/bootstrap-vue/esm/components/table/helpers/default-sort-compare.js 68:49-70
export 'stringifyObjectValues' (imported as 'stringifyObjectValues') was not found in '../../../utils/stringify-object-values' (possible exports: default)
 @ ./node_modules/bootstrap-vue/esm/components/table/helpers/mixin-sorting.js 17:0-60 98:21-39
 @ ./node_modules/bootstrap-vue/esm/components/table/table.js 25:0-78 33:471-483 43:32-44
 @ ./node_modules/bootstrap-vue/esm/index.js 249:0-50 249:0-50
 @ ./src/app.js 3:0-58 8:8-20 10:8-19

WARNING in ./node_modules/bootstrap-vue/esm/components/table/helpers/mixin-filtering.js 263:27-48
export 'stringifyRecordValues' (imported as 'stringifyRecordValues') was not found in './stringify-record-values' (possible exports: default)
 @ ./node_modules/bootstrap-vue/esm/components/table/table.js 20:0-84 33:389-403 43:16-30
 @ ./node_modules/bootstrap-vue/esm/index.js 249:0-50 249:0-50
 @ ./src/app.js 3:0-58 8:8-20 10:8-19

webpack 5.36.0 compiled with 3 warnings in 8904 ms

### Steps to reproduce the bug

1. Create a vue project without using the vue-cli service and use Wepack instead
2. Start the dev server 
3. See error or warnings

### Expected behavior

It solves changing the var to default key word in stringifyObjectValues definition and the same for stringifyRecordValues.
Basically I changed as the warning told me to. I'll show you:

export var stringifyObjectValues = function stringifyObjectValues(value) {} **to**
export default stringifyObjectValues = function stringifyObjectValues(value) {}

For **stringifyObjectValues** in this path:
./node_modules/bootstrap-vue/esm/components/table/helpers/default-sort-compare.js

For **stringifyRecordValues** in this path:
./node_modules/bootstrap-vue/esm/components/table/helpers/mixin-filtering.js

### Versions

**Libraries:**

- BootstrapVue: 2.21.2
- Bootstrap: 4.6.0
- Vue: 2.6.11

**Environment:**

- Device: [Laptop Hp]
- OS: [Ubuntu 20.0.4]
- Browser: [Firefox Developer Edition]

**Comment by Hiws** [comment]
Looks like it has something to do with how the ESM build is made, as the source code both uses `export const = () => {}`.

https://github.com/bootstrap-vue/bootstrap-vue/blob/7d62956d46070ac935cb0266b540979b35032f56/src/utils/stringify-object-values.js#L17-L31

https://github.com/bootstrap-vue/bootstrap-vue/blob/7d62956d46070ac935cb0266b540979b35032f56/src/components/table/helpers/stringify-record-values.js#L7-L11
